README.md

```diff
@@ -557,6 +557,14 @@ $job = $client->jobs()->show($jobId);
 ```
 Returns the job.
 
+##### Wait for a job to finish
+```php
+$numberOfMinutesToWait = 3;
+$jobHelper = new \PrivatePackagist\ApiClient\JobHelper($client);
+$job = $jobHelper->waitForJob($jobId, $numberOfMinutesToWait);
+```
+Returns the job.
```

Comment:
Document that this throws an exception and what kind of there is an error or a timeout and show how to handle them.